package.json

```diff
@@ -6,9 +6,11 @@
     "test": "pulp test"
   },
   "devDependencies": {
-    "pulp": "^9.0.1",
-    "purescript": "^0.9.1",
-    "purescript-psa": "^0.3.9",
     "rimraf": "^2.5.0"
+  },
+  "dependencies": {
+    "pulp": "^9.0.1",
+    "purescript": "^0.10.1",
+    "purescript-psa": "^0.3.9"
```

Comment:
I think these should remain `devDependencies` since they're for building it... I guess it doesn't really matter since we don't distribute via npm anyway!

Something that might help: I use a package from `npm` called `node-check-updates`, if you install that globally you can just run this:

``` 
ncu -a
ncu -m bower -a
```

To update all the existing dependencies for both `npm` and `bower` without having to do it manually.